packages/flame_markdown/example/.metadata

```diff
@@ -0,0 +1,30 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled and should not be manually edited.
```

Comment:
should it though, flutter? I always feel like it is a mistake to commit this file as it keeps changing when building on different setups

Reply:
It should only be version controlled when it is like an app that is build in a pipeline, so that it needs to be possible to replicate the exact built.
https://dart.dev/guides/libraries/private-files#pubspeclock

Reply:
Ooh, this was the .metadata file, I'm way too tired clearly :sweat_smile:

Reply:
been there!